data/ansible/domain.yml

```diff
@@ -12,3 +12,15 @@ slots: {}
 responses:
   utter_ansible_inform_reach_out_to_support:
     - text: "I'm sorry you are having trouble accessing a namespace. Please reach out to [partner-help@redhat.com](mailto:partner-help@redhat.com) for any questions about the namespace access."
+  utter_ansible_inform_assist_with_free_trial:
```

Comment:
Can you send these as multiple messages instead of having newlines in each? The UI renders those differently. Like:

```
utter_ansible_inform_assist_with_free_trial_1:
   - text: blah
utter_ansible_inform_assist_with_free_trial_2:
   - text: blah2
```